src/util/hass-util.html

```diff
@@ -47,16 +47,20 @@
   return hass.reactor.evaluate(hass.serviceGetters.canToggleEntity(entityId));
 };
 
+// Update root's child element to be newElementTag replacing another existing child if any.
+// Copy attributes into the child element.
+// If newElementTag is null just update the existing child element if any.
 window.hassUtil.dynamicContentUpdater = function (root, newElementTag, attributes) {
   var rootEl = Polymer.dom(root);
   var customEl;
 
-  if (rootEl.lastChild && rootEl.lastChild.tagName === newElementTag) {
+  if (rootEl.lastChild && (rootEl.lastChild.tagName === newElementTag || !newElementTag)) {
```

Comment:
Not sure if I see what is happening here? We should only assign customEl to lastChild if it is the same tag.